packages/core/src/Site/template.ts

```diff
@@ -1,17 +1,39 @@
 import fs from 'fs-extra';
 import path from 'path';
+import walkSync from 'walk-sync';
 import * as fsUtil from '../utils/fsUtil';
+import { INDEX_MARKDOWN_FILE, SITE_CONFIG_NAME, _ } from './constants';
+import { SiteConfig, SiteConfigPage } from './SiteConfig';
+import { VariableRenderer } from '../variables/VariableRenderer';
+import * as logger from '../utils/logger';
+
+const { LAYOUT_DEFAULT_NAME, LAYOUT_FOLDER_PATH } = require('../Layout');
 
 const requiredFiles = ['index.md', 'site.json', '_markbind/'];
 
 const PATH_TO_TEMPLATE = '../../template';
+const ABOUT_MARKDOWN_FILE = 'about.md';
+const ABOUT_MARKDOWN_DEFAULT = '# About\n'
+  + 'Welcome to your **About Us** page.\n';
+const CONFIG_FOLDER_NAME = '_markbind';
+const SITE_FOLDER_NAME = '_site';
+const WIKI_SITE_NAV_PATH = '_Sidebar.md';
+const WIKI_FOOTER_PATH = '_Footer.md';
+
+type NaviagablePage = {
+  src: string,
+  title?: string,
+};
 
 export class Template {
-  root: string;
+  rootPath: string;
   template: string;
```

Comment:
If we are changing `root` back to `rootPath`, then we may as well also change back `template` to `templatePath`?